src/camera.ts

```diff
@@ -259,17 +293,29 @@ class Camera extends Element {
 
         // in with the new
         const pixelFormat = PIXELFORMAT_RGBA8;
+        const samples = this.scene.config.camera.multisample ? device.maxSamples : 1;
```

Comment:
any benefits to use multisampling in this tool?

Reply:
nope, it's disabled in config https://github.com/playcanvas/super-splat/blob/main/src/scene-config.ts#L23, but I decided to keep support in the code for now.